components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -4952,20 +4950,15 @@ public static String resolveExternalConsentPageUrl(String tenantDomain) throws I
      * @return Whether the application should be FAPI conformant.
      * @throws IdentityOAuth2Exception
```

Comment:
Update comment mentioning the new exception being thrown

Reply:
Updated